### PR TITLE
Adding containers array to request and response object, to handle container activity logs

### PR DESCRIPTION
**Pull request by pladam**
Adding containers array to request and response object, to handle container activity logs

1. This change adds `group` as an `optional` to api object

2. This change adds `containers[]` as an `optional` to api.request
3. This change adds `data' as an `optional` to api.request
4. This change adds `containers[]` as an `optional` to api.response
5. This change adds `data' as an `optional` to api.response

6. This change adds `namespace' as an `optional` to resource_details
7. This change adds `log_level` as an `optional` to metadata object

8. This change adds `log_level` to dictionary
9. This change adds `containers[]` to dictionary

### CODE REVIEW

File: objects/response.json [hunk 1]
```diff
@@ -7,6 +7,14 @@
     "code": {
       "requirement": "recommended"
     },
+    "containers": {
```
Comment (rroupski) [inline objects/response.json:10]: The default description of the `containers` attribute is too generic, perhaps add a specific one to explain what is the rule/usage of the `containers` in the response object.

File: objects/request.json [hunk 2]
```diff
@@ -4,6 +4,14 @@
   "extends": "object",
   "name": "request",
   "attributes": {
+    "containers": {
```
Comment (rroupski) [inline objects/request.json:7]: The default description of the `containers` attribute is too generic, perhaps add a specific one to explain what is the rule/usage of the `containers` in the request object.

**Review by rroupski — CHANGES_REQUESTED** [review]

File: objects/response.json [hunk 3]
```diff
@@ -7,6 +7,14 @@
...
+      "requirement": "optional",
+      "profile": "container"
```
[the remark this answers is not in the document]
Comment (pladam) [reply objects/response.json]: updated.

**Review by rroupski — APPROVED** [review]

File: objects/resource_details.json [hunk 4]
```diff
@@ -16,6 +16,10 @@
       "description": "The name of the related resource group.",
       "requirement": "optional"
     },
+    "namespace": {
+      "description": "The namespace is useful when similar entities exists that you need to keep separate.",
```
Comment (Aniak5) [inline objects/resource_details.json:20]: I think that "exists" should be "exist".

**Review by mr-splunk — DISMISSED** [review]

**Review by Aniak5 — APPROVED** [review]
LGTM

**Review by mr-splunk — APPROVED** [review]

**Review by irakledibm — APPROVED** [review]

**Review by rroupski — APPROVED** [review]

**Pull request merged**